dll/win32/syssetup/wizard.c

```diff
@@ -2968,6 +2970,29 @@ ProcessSetupInf(
     }
 #endif
 
+    *szValue = UNICODE_NULL;
+    pData = szValue;
+
+    GetWindowsDirectoryW(szValue, ARRAYSIZE(szValue));
+    StringCchCatW(szValue, ARRAYSIZE(szValue), L"\\inf");
+
+    if (FAILED(StringCchLengthW(szValue, ARRAYSIZE(szValue), &cchLength)))
+    {
+        return;
+    }
+
+    pData += cchLength + 1;
+    cchLength = (ARRAYSIZE(szValue) - cchLength - 1);
+    StringCchCopyW(pData, cchLength, pSetupData->SourcePath);
+
+    if (FAILED(StringCchLengthW(pData, ARRAYSIZE(szValue), &cchLength)))
```

Comment:
So it should be `The max number of characters allowed in  pData `` , but since pData is not pointing anymore to the beginning of the szValue buffer, that length value should be reduced accordingly as well.

Reply:
Fixed.